charts/kargo/values.yaml

```diff
@@ -377,6 +377,21 @@ controller:
     ## @param controller.globalCredentials.namespaces List of namespaces to look for shared credentials. Note that as of v1.0.0, the Kargo controller does not have cluster-wide access to Secrets. The controller receives read-only permission for Secrets on a per-Project basis as Projects are created. If you designate some namespaces as homes for "global" credentials, you will need to manually grant the controller permission to read Secrets in those namespaces.
     namespaces: []
 
+  ## Reconciler-specific settings
+  reconcilers:
+    controlFlowStages:
+      ## @param controller.reconcilers.controlFlowStages.maxConcurrentReconciles specifies the maximum number of control flow Stage resources the controller can reconcile concurrently.
+      maxConcurrentReconciles: 4
+    promotions:
+      ## @param controller.reconcilers.promotions.maxConcurrentReconciles specifies the maximum number of Promotion resources the controller can reconcile concurrently.
+      maxConcurrentReconciles: 4
+    stages:
+      ## @param controller.reconcilers.stages.maxConcurrentReconciles specifies the maximum number of (non-control flow) Stage resources the controller can reconcile concurrently.
+      maxConcurrentReconciles: 4
+    warehouses:
+      ## @param controller.reconcilers.warehouses.maxConcurrentReconciles specifies the maximum number of Warehouse resources the controller can reconcile concurrently.
+      maxConcurrentReconciles: 4
```

Comment:
Would it be an idea to have a default fallback, in case you just want to scale lazily?

Reply:
Not sure I follow? It defaults to 4 in the code as well. I debated leaving it undefined here, but decided this was more clear.

Reply:
I mean something like: `{{ default .Values.reconcilers.global.maxConcurrentReconciles .Values.reconcilers.stages.maxConcurrentReconciles }}`. Changing the global value would cause all values to change _unless_ they have a kind specific value set.

Reply:
Not to keen on the `.global` here by the way, but you get the point.

Reply:
I get your meaning now. I can do something like that without the nasty global bit.

Reply:
Addressed in f7c2294bfc357a01fb84dde318adb1aebb231f22